tests/tests_suite_test.go

```diff
@@ -121,14 +121,10 @@ var _ = AfterEach(func() {
 })
 
 var _ = AfterSuite(func() {
-	cancelUserSess, cancelUserErr := cancelSess(url, testUser, testPassword)
-	cancelAdminSess, cancelAdminErr := cancelSess(url, testAdminUser, testAdminPassword)
+	os.Chdir(testHome)
```

Comment:
This is necessary to avoid the madness of not having a directory structure to work in:

```
Logged in as test-81
•SSSSSSSSSSSSSSSSSSSSSSSSSshell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
Logged in as test-81
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
Please log in again in order to cancel this account
Logged in as test-81
Account cancelled
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
Logged in as admin
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
Please log in again in order to cancel this account
Logged in as admin
Account cancelled
```